packages/form/addon/components/cf-field/input/date.js

```diff
@@ -13,22 +15,17 @@ export default class CfFieldInputDateComponent extends Component {
     return this.intl.primaryLocale.split("-")[0];
   }
 
-  /*
-   * Extract the date format string for the current locale.
-   * For example en-us will return "m/d/Y" and de-ch will return "d.m.Y".
-   */
   get dateFormat() {
-    const sample = this.intl.formatDate(
-      DateTime.fromISO("1970-12-31").toJSDate(),
-      {
-        day: "2-digit",
-        month: "2-digit",
-        year: "numeric",
-      }
-    );
-
-    // refer to flatpickr documentation for formatting options
-    return sample.replace(31, "d").replace(12, "m").replace(1970, "Y");
+    const config = getOwner(this).resolveRegistration("config:environment");
```

Comment:
I would move the config resolving in a separate getter, since it doesn't change along with the locale.
```
get config() {
    return getOwner(this).resolveRegistration("config:environment");
  }
```

Reply:
Done